src/main/kotlin/com/jaredsburrows/spoon/SpoonExtension.kt

```diff
@@ -34,6 +34,9 @@ open class SpoonExtension { // Extensions cannot be final
     /** Path to output directory. ("$buildDir/spoon-output" by default) */
     var output: String = DEFAULT_OUTPUT_DIRECTORY
 
+    /** Final path to output directory. Need to avoid cross-variant execution conflicts */
+    lateinit var finalOutput: String
```

Comment:
Can we use update and re-use the `output` variable I already had?

Also, I rather just have a default value instead of `lateinit`, this is just supposed to be a "dummy" model class.

Reply:
Also, where is the `output` var being used.

Can you take a look at this solution here: https://github.com/stanfy/spoon-gradle-plugin/blob/master/src/main/groovy/com/stanfy/spoon/gradle/SpoonPlugin.groovy#L114.